src/components/ProductList/ItemsList/ItemsListContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import ItemsList from './ItemsList';
+import { getProductsSuccess, getProductsError } from '../../../actions/actionsProductList';
+
+const mapStateToProps = (state) => ({
+  ascendingOrder: state.reducerProductList.ascendingOrder,
+  error: state.reducerProductList.error,
+  itemsOnPage: Number(state.reducerProductList.chosenItemsOnPage),
```

Comment:
Have you tried **parseInt** ?

Reply:
I need to use `Number` here, because I expect it to return NaN in case `chosenItemsOnPage` is not a number. The `parseInt` parses up to first non-digit and returns whatever value has been parsed, that's not exactly what I need in this case.

Reply:
Maybe, but why not use 0 as default value instead?